CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ All notable changes to this project will be documented in this file.
 
 - Add RT-DETR model for object detection task
   (https://github.com/openvinotoolkit/training_extensions/pull/3741)
+- Add Multi-Label & H-label Classification with torchvision models by @harimkang (https://github.com/openvinotoolkit/training_extensions/pull/3697)
```

Comment:
Referring to previous versions' logs, an author for PR is omitted.

Reply:
Thanks, updated it - https://github.com/openvinotoolkit/training_extensions/pull/3768/commits/cba2f781114e5e780058b4da0582bc0d8bd68c4b